cli/utils/common.go

```diff
@@ -1,9 +1,13 @@
 package utils
 
 import (
+	"fmt"
 	"io"
+	URL "net/url"
```

Comment:
this is not a very standard thing to do in go. we usually just use the package name directly, specially with stdlib packages. In the case that you have a conflicting var name like in the `OpenBrowser` func, just rename the variable. In this kind of short funcs, you could just use `u string` for the var name, or `rawURL` to indicate that it's an unparsed URL, just as quick examples